internal/clearingway/guild.go

```diff
@@ -39,6 +43,9 @@ type Guild struct {
 	UltimateRepetitionRoles *Roles
 	DatacenterRoles         *Roles
 	AchievementRoles        *Roles
+	MenuRoles               *Roles
```

Comment:
not really sure what the purpose of `MenuRoles` is

Reply:
The purpose of this is to ensure any additional roles added by the menu (mainly relevant in #34) is ensured on startup

Reply:
we should leave a comment for this then